kinto_attachment/__init__.py

```diff
@@ -23,10 +23,20 @@ def includeme(config):
 
     # Advertise public setting.
     config.registry.public_settings.add('attachment.base_url')
+    config.registry.public_settings.add('attachment.extra.base_url')
```

Comment:
Otherwise what do you think of exposing only one setting?
With something like:
```py
base_url = settings.get('attachment.extra.base_url', settings.get('attachment.base_url'))
config.registry.public_settings.add(base_url)
```